Wrappers/Python/test/test_cropped_readers.py

```diff
@@ -0,0 +1,78 @@
+import os
+import unittest
+
+import numpy as np
+import vtk
+from ccpi.viewer.utils.conversion import (Converter, cilRawCroppedReader,
+                                          cilMetaImageCroppedReader,
+                                          cilNumpyCroppedReader)
+
+
+
+class TestCroppedReaders(unittest.TestCase):
+
+    def setUp(self):
+        # Generate random 3D array and write to HDF5:
+        self.input_3D_array = np.random.randint(10, size=(5, 10, 6), dtype=np.uint8)
```

Comment:
it's not quite so relevant in this PR, but normally when using "random" numbers one would actually seed the generator so that the random numbers (in the unit test) are always the same. This is useful when checking values.